chapter08/spg/exercise1.pl

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+my $file;
+my $string;
+
+print "Enter 1) for scalar filehandle, or 2) for file filehandle\n";
+
+chomp ( my $input = <STDIN> );
+
+if ( $input == 1 ) {
+  open $file, '>', \ $string;
+}
+elsif ( $input == 2 ) {
+  open $file, '>', 'date.out' or die "Could not open date.out: $!";
+}
+else { 
+  print "Wrong move, chump";
+  die;
+}
+
+my $date = localtime;
+my $dow = (localtime)[6];
+
+print $file "Date is: ", $date, " and day of week is: ", $dow;

```

Comment:
Isn't the day of the week here just an integer? It would be a bit ambiguous to someone not familiar with the crazy that is `localtime`